CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # Change Log
 
 ## [0.7.2](https://github.com/maxpoint/conda-mirror/tree/0.7.2) (2017-05-23)
-
```

Comment:
Do you still want this blank line? Not sure how you like to do formatting here. So this is merely a question.